From 931b70e6575933f03202df639a2a72506f9b73ac Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 7 Aug 2022 16:56:39 +0200
Subject: [PATCH] meson.build: add libinput dependency

this would fix:
ld: clay.p/src_clay_input.c.o: undefined reference to symbol 'libinput_device_config_scroll_has_natural_scroll@@LIBINPUT_0.12.0'

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index dfcb0e1..50ddf93 100644
--- a/meson.build
+++ b/meson.build
@@ -32,6 +32,7 @@ wayland_server = dependency('wayland-server')
 wayland_client = dependency('wayland-client')
 pixman         = dependency('pixman-1')
 xkbcommon      = dependency('xkbcommon')
+libinput       = dependency('libinput')
 math           = cc.find_library('m')
 
 wl_protocol_dir = wayland_protos.get_variable('pkgdatadir')
@@ -101,6 +102,7 @@ executable(
     wlroots,
     xkbcommon,
     pixman,
+    libinput,
     math,
   ],
   install: true,
-- 
2.34.1

